src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/put_file_request.cc

```diff
@@ -164,6 +164,7 @@ void PutFileRequest::Run() {
   is_persistent_file_ = false;
   bool is_system_file = false;
   length_ = binary_data.size();
+  // cppcheck-suppress variableScope
   bool is_download_complete = true;
```

Comment:
Fix variable scope

Reply:
Fixed in 86dbcbdcf96ad8b7cf805c4ca90b57b1a347178b
Please check